### DISCUSSION TITLE
where is "t" defined in the cadquery-cycloidal example? (noob question)

### DISCUSSION THREAD

**Discussion by ht55ght55** [sgenoud/replicad#20] [Q&A]
I'm trying to understand the logic and I probably can't see something obvious.  Please help!

<img width="624" alt="image" src="https://user-images.githubusercontent.com/52102108/172257083-63211b66-42ec-4ddb-8e20-f0804b670bf1.png">

**Comment by sgenoud** [comment]
The t is just an argument of the cycloidal functions (I am using [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)).

The value of the parameter `t` is then filled by the `sketchParametricFunction`, at different points to create an interpolation.
